spec/policies/portfolio_item_policy_spec.rb

```diff
@@ -139,6 +152,32 @@
         end
       end
     end
+
+    context "when there are no parameters at all" do
+      let(:user_context) { UserContext.new("current_request", nil, "controller_name") }
```

Comment:
@eclarizio do we need a controller_name here?

Reply:
We don't, but I didn't adjust the `UserContext` class at all, so I left it as-is for now.